src/args.rs

```diff
@@ -1,8 +1,125 @@
-#[derive(Debug)]
-pub struct Args {}
+use clap::Parser;
+
+#[derive(Parser, Debug, Default)]
+#[command(author, version, about, long_about = None, name = "jperf")]
+pub struct Args {
+	/// Application name
+	#[arg(long)]
+	app: Option<String>,
+
+	/// Cloud name
+	#[arg(long)]
+	cloud: Option<String>,
+
+	/// Unit name
+	#[arg(long)]
+	unit: Option<String>,
+
+	/// Model name
+	#[arg(long)]
+	model: Option<String>,		
+
+	/// Display a large set of metrics
+	#[arg(short, long)]
+	verbose: bool,
+
+	/// Optional choice of a single metric to display
+	#[arg(short, long)]
+	metric: Option<String>,	
```

Comment:
I think mentioning that `metric` is optional is redundant here, so perhaps this doc string should just follow the convention of the other filtering options. Also, this option has both a long and a short form, let's just keep the long form as I don't think this will be specified extremely frequently

Reply:
Makes sense! Changed.